package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "start": "./bin/server.js",
-    "test": "jshint *.js **/*.js && jscs *.js **/*.js && nodeunit"
+    "test": "jshint {.,bin,lib,lib/repo,test}/*.js && jscs {.,bin,lib,lib/repo,test}/*.js && nodeunit"
```

Comment:
I've tried both `"excludeFiles": []` and `"excludeFiles": ["node_modules/"]`, along with `jscs .` (jscs 2.1.1), neither option works, it just validates everything, which takes forever and I have to kill the process.